### PR TITLE
#163537823: Some of the UI pages redesigned.

### PR DESCRIPTION
**Pull request by kenzdozz**
#### What does this PR do?
Some of the UI pages redesigned.
#### Description of Task to be completed?
- Redesiged of the homepage.
- Added required form input on register page.
- Added party headquarters address on parties page.
- Moved Express interest to main menu.
- Included mandate on express interest pop up.
- Fixed colors on admin dashboard.
#### How should this be manually tested?
Can be seen by going through the UI pages
#### What are the relevant pivotal tracker stories?
[#163537823](https://www.pivotaltracker.com/story/show/163537823/comments/199023366)

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -52,20 +83,8 @@ main {
...
-}
-.slide.show {
-    display: block;
-}
-.slide.show img{
-    animation: rollIn 2.5s linear;
-}
-.slide.show h2{
-    animation: fadeInOut 2.5s linear;
 }
```
Comment (ascii-dev) [inline css/home.css]: A new line here, please

File: index.html [hunk 2]
```diff
@@ -11,30 +11,38 @@
...
-        <p class="text-right w-100 pr-3">&copy; <span id="copyYear">2019</span> Politico</p>
     </main>
-    <script src="./js/script.js"></script>
-    <script src="./js/home.js"></script>
+    <footer>
+        <p class="text-center">&copy; <span id="copyYear">2019</span> Politico</p>
+    </footer>
 </body>
 
 </html>
```
Comment (ascii-dev) [inline index.html]: A new line here, please

File: css/home.css (hunk 1 above)
Comment (kenzdozz) [reply css/home.css]: New line added

File: index.html (hunk 2 above)
Comment (kenzdozz) [reply index.html]: New line added

**Pull request merged**